todo/index.html

```diff
@@ -7,16 +7,26 @@
     </head>
     
     <body>
-        <h1 class="title"  >ToDoList</h1>
+        <h1 class="title"  >TODO LIST</h1>
 
         <div class="card">
-            <input class="text-todo" type="text"  placeholder="Add your ToDO">
-            <button class="button-plus">
-                <img class="button-plus-image" src="img/1200px-Plus_symbol.svg.png">
-            </button>
-            <input id="search" class="text-search" type="text"  placeholder="Search">
+            <input id="adding" class="text-todo" type="text"  placeholder="Add your ToDO" style='border: 0px; '>
```

Comment:
It's bad practice to use inline styles. Prease replace that to style.css
And you should create more meaningful names for identifications, classes, variables. For example, newTask for current ID.